src/firecracker/swagger/firecracker.yaml

```diff
@@ -1269,6 +1236,11 @@ definitions:
         type: boolean
         description:
           When set to true, the vm is also resumed if the snapshot load is successful.
+      shared:
```

Comment:
Should this be called `mmap` because firecracker has a thing about using `userfaultfd`? I was originally going to suggest using `async` but that would clash with `userfaultfd`.

Reply:
It's `mmap` in both cases, whether it's true or false! `shared` is what is used internally in upstream Firecracker for `MAP_ASYNC` - they introduced it sometime in ~1.6 IIRC

Reply:
Oh so then let's call it `async`!

Reply:
Sorry, I meant `MAP_SHARED` here, no such thing as `MAP_ASYNC` - `MAP_ANONYMOUS` is the incerse ^^ That's the actual flag name and why `shared` is used: https://man7.org/linux/man-pages/man2/mmap.2.html

Reply:
Ah okay that makes sense - let's keep it as is then.